src/modules/collectscores.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-restricted-globals */
+import add from './addname.js';
```

Comment:
- [ ] Let's try to not disable Linter checks in our projects. Linter checks help us write better and more professional code and disabling it defies its purpose. Kindly fix the error 👍🏽